support-workers/build.sbt

```diff
@@ -69,7 +69,7 @@ assemblyMergeStrategy in (IntegrationTest, assembly) := {
     oldStrategy(y)
 }
 IntegrationTest / assembly / test := {}
-IntegrationTest / assembly / aggregate := false
+assembly / aggregate := false
```

Comment:
this is quite useful to apply to all assembly, this means you can run `support-workers / assembly` in sbt and not get all the sub-jars pulled in